tests/test_accessory.py

```diff
@@ -9,39 +9,50 @@
 class TestAccessory(object):
 
     def test_init(self):
-        acc = accessory.Accessory("Test Accessory")
+        acc = accessory.Accessory('Test Accessory')
 
     def test_publish_no_broker(self):
-        acc = accessory.Accessory("Test Accessory")
-        service = loader.get_serv_loader().get_service("TemperatureSensor")
-        char = service.get_characteristic("CurrentTemperature")
+        acc = accessory.Accessory('Test Accessory')
+        service = loader.get_serv_loader().get_service('TemperatureSensor')
+        char = service.get_characteristic('CurrentTemperature')
         acc.add_service(service)
         char.set_value(25, should_notify=True)
 
+    def test_set_services_compatible(self):
+        """Test that _set_services still works and has access to the info services"""
+        class A(accessory.Accessory):
+            def _set_services(self):
+                super()._set_services()
+                s = loader.get_serv_loader().get_service("TemperatureSensor")
+                self.add_service(s)
+                assert(self.get_service("AccessoryInformation") is not None)
+        a = A("Test Accessory")
+        assert(a.get_service("TemperatureSensor") is not None)
```

Comment:
Same here

Reply:
Thanks! Writing cpp during the day and python at night/weekend really messes up your coding habits